### PR TITLE
NR-174996 remove dependence from amqp lib for integration tests

### PR DESCRIPTION
**Pull request by rajrohanyadav**
### What?
Remove `github.com/streadway/amqp` library from requirements.

### Why?
This lib was used to just perform health_check for the RabbitMQ service on the container. Doesn't make sense to have this dependency for this task. Rather we can use the commandline utility to perform healthchecks.

### CODE REVIEW

File: tests/helper.go [hunk 1]
```diff
@@ -9,11 +9,43 @@ import (
...
+	if err != nil {
+		return "", "", err
+	}
+
+	stdout := outbuf.String()
+	stderr := errbuf.String()
+	return stdout, stderr, nil
+}
+
+func waitForRabbitMQIsUpAndRunning(maxTries int, containerName string) bool {
```
Comment (paologallinaharbur) [inline tests/helper.go:35]: why:
```golang
	envVars := []string{}
	ports := []string{"5672:5672", "15672:15672"}
	stdout, stderr, err := dockerComposeRunMode(envVars, ports, containerRabbitMQ, true)
	if err != nil {
		log.Fatal(err)
	}
	fmt.Println(stdout)
	fmt.Println(stderr)
```
is no longer part of this function?

**Review by paologallinaharbur — DISMISSED** [review]
Thanks for taking care of this!

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 Unreleased section should follow [Release Toolkit](https://github.com/newrelic/release-toolkit#render-markdown-and-update-markdown)
 ## Unreleased
 
+### enhancement
+- Remove dependence on amqp library for integration tests
+
```
Comment (gsanchezgavier) [inline CHANGELOG.md:13]: this is going to bump the minor version of the integration IIRC. I would not even add a changelog entry in this particular PR perhaps.

Comment (rajrohanyadav) [reply CHANGELOG.md:13]: Makes sense. Nothing changes in source code. Thanks.

Comment (rajrohanyadav) [reply CHANGELOG.md]: done.

**Review by gsanchezgavier — APPROVED** [review]
thanks!

**Review by paologallinaharbur — APPROVED** [review]

**Pull request merged**